src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -68,10 +68,15 @@ class ChannelListMemberSchema implements BaseDbSchema {
   };
 
   saveIfNotExist = async (db: SQLiteDatabase): Promise<void> => {
-    const ifExists = await this.checkIfExist(db);
-    if (ifExists) return;
+    try {
+      const ifExists = await this.checkIfExist(db);
+      if (ifExists) return;
 
-    this.save(db);
+      this.save(db);
+    } catch (e) {
+      console.log('saveIfNotExist error', e);
+      throw e;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `saveIfNotExist` function is a good addition. However, logging errors to the console isn't always the best practice for production code. It would be better to use a dedicated logging service that can provide more context and control over what gets logged and where those logs are sent. If you don't have a logging service, consider setting one up.

```diff
-       console.log('saveIfNotExist error', e);
+       Logger.error('saveIfNotExist error', e);
```
Please replace `console.log` with your preferred logging method. This will help you keep track of errors in a more structured way.


<!-- This is an auto-generated comment by OSS CodeRabbit -->